release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2093,6 +2093,8 @@ To make the required change, follow the instructions in link:https://access.redh
 +
 (link:https://issues.redhat.com/browse/OCPBUGS-15997[*OCPBUGS-15997*])
 
+* Broadcom network interface controllers in legacy Single Root I/O Virtualization (SR-IOV) do not support quality of service (QoS) and tag protocol identifier (TPID) settings for the SRIOV VLAN. This affects Broadcom_BCM57414, Broadcom_BCM57508, and Broadcom_BCM57504. (link:https://issues.redhat.com/browse/RHEL-9881[*RHEL-9881*])
```

Comment:
I would remove the _ from Broadcom_BCM**, so it would be "Broadcom BCM57414" for example.